target/linux/kirkwood/image/Makefile

```diff
@@ -184,6 +184,25 @@ define Device/iom_ix2-200
 endef
 TARGET_DEVICES += iom_ix2-200
 
+define Device/iom_ix4-200d
+  DEVICE_VENDOR := Iomega
+  DEVICE_MODEL := StorCenter ix4-200d
+  DEVICE_PACKAGES := kmod-ata-marvell-sata kmod-fs-ext4 \
+	kmod-gpio-button-hotplug kmod-mvsdio kmod-hwmon-adt7475 \
+	kmod-spi-gpio kmod-gpio-nxp-74hc164
+  SUPPORTED_DEVICES += marvell,rd88f6281-a
```

Comment:
why?

Reply:
Why `SUPPORTED_DEVICES ` is added? It is used for legacy purposes.

Reply:
And that is wrong.  Please clean this https://github.com/openwrt/openwrt/pull/12543/files#r1187088402 and `SUPPORTED_DEVICES ` line. First sysupgrade must be with `-F` and next should be work normal.